ubuntu/Dockerfile

```diff
@@ -41,5 +45,5 @@ RUN cd /tmp \
  && cmake --build . --target install \
  && rm -rf /tmp/* /usr/src/root
 
-ENV PYTHONPATH /usr/local/lib
+ENV PYTHONPATH /usr/local/lib:/usr/local/lib/root
```

Comment:
Is this needed now that you added `-Dgnuinstall=OFF`? I guess not.

Reply:
I don't know, do you want me to test?

Reply:
I can test the final `Dockerfile`, but only when `gnuinstall=ON` libraries go to `lib/root` subdir, so this can be removed in case we use `gnuinstall=OFF`.